app/js/mol.map.layers.js

```
@@ -72,21 +72,33 @@ mol.modules.map.layers = function(mol) {
                         _.each(
                             event.layers,
                             function(layer) {
-                                var extent = $.parseJSON(layer.extent);
-                                var layer_bounds = new google.maps.LatLngBounds(
-                                        new google.maps.LatLng(extent.sw.lat,extent.sw.lng),
-                                        new google.maps.LatLng(extent.ne.lat,extent.ne.lng)
-                                     );
-                                if(!bounds) {
-                                    bounds = layer_bounds;
-                                } else {
-                                    bounds.union(layer_bounds)
-                                }
+                                var extent,
+                                    layer_bounds;
+                                try {
+                                    extent = $.parseJSON(layer.extent);
+                                    layer_bounds = new google.maps.LatLngBounds(
+                                        new google.maps.LatLng(
+                                            extent.sw.lat,extent.sw.lng
+                                        ),
+                                        new google.maps.LatLng(
+                                            extent.ne.lat,extent.ne.lng
+                                        )
+                                    );
+                                    if(!bounds) {
+                                        bounds = layer_bounds;
+                                    } else {
+                                        bounds.union(layer_bounds)
+                                    }
+                                } 
+                                catch(e) {
+                                    //bad json
```

We might want to log this to the console instead of silently ignoring it.